qupath-extension-bioformats/src/main/java/qupath/lib/images/writers/ome/ConvertCommand.java

```diff
@@ -126,8 +127,8 @@ public void run() {
 		}
 		
 		String[] args;
-		if (series > 0)
-			args = new String[]{"--series", Integer.toString(series)};
+		if (series >= 0)
```

Comment:
Ah yes, looks like it will. The intention was to make it possible to force the use of Bio-Formats as the reader (since it's the only one that uses the series), but I think you're right - it'll take more than this.

Reply:
If you set the default to be -1, then this code will work.

Reply:
(`series` isn't used anywhere else in this Java file.)